R/derive_param_bor.R

```diff
@@ -1,5 +1,9 @@
 #' Adds a Parameter for Best Overall Response (without confirmation)
 #'
+#' @description
+#' `r lifecycle::badge("superseded")` The `derive_param_bor()` function has been
+#' superseded in favor of `derive_extreme_event()`.
```

Comment:
This function is using filter_pd(). Do we want to mention that filter_pd() function was superseded in favor of filter_relative()?

Reply:
I don't think we need to mention it because it is of interest for developers only.